### PR TITLE
PyPortal_Quarantine_Clock: update to settings.toml

### PR DESCRIPTION
**Pull request by dhalbert**
Guide also updated.

### CODE REVIEW

File: PyPortal/PyPortal_Quarantine_Clock/code.py [hunk 1]
```diff
@@ -88,7 +82,7 @@
     if (not refresh_time) or (time.monotonic() - refresh_time) > 3600:
         try:
             print("Getting new time from internet...")
-            pyportal.get_local_time(secrets['timezone'])
+            pyportal.get_local_time(os.getenv("TIMEZONE"))
```
Comment (brentru) [inline PyPortal/PyPortal_Quarantine_Clock/code.py:85]: The timezone was previously lower case "timezone". 

In the guide - Did it change to "TIMEZONE" in the toml file?

**Review by brentru — CHANGES_REQUESTED** [review]

File: PyPortal/PyPortal_Quarantine_Clock/code.py (hunk 1 above)
Comment (dhalbert) [reply PyPortal/PyPortal_Quarantine_Clock/code.py:85]: Yup, guide is all upper-case entries in settings.toml now.

**Comment by dhalbert** [comment]
The build failure is in an unrelated file. Ready to re-review.

**Review by TheKitty — APPROVED** [review]
image
You defined OS but it's not finding. Approved

**Comment by dhalbert** [comment]
Yeah, that error  is in TOTP_Friend, which I forgot to say "Make a new branch on". I fixed that, but it is running the PR tests against the pre-fixed version. Problem will go away on merge.

**Pull request merged**